src/pal/src/misc/time.cpp

```diff
@@ -270,6 +275,21 @@ GetTickCount64()
         }
         retval = (ts.tv_sec * tccSecondsToMillieSeconds)+(ts.tv_nsec / tccMillieSecondsToNanoSeconds);
     }
+#elif HAVE_MACH_CLOCK_MONOTONIC
+    {
+        kern_return_t machRet;
+        clock_serv_t clock;
+        mach_timespec_t mts;
+
+        host_get_clock_service(mach_host_self(), SYSTEM_CLOCK, &clock);
```

Comment:
Yes. I missed checking the return value here. But I am deallocating the host port using 'mach_port_deallocate' a bunch of lines below. I need to make it better to deallocate the host port even in case of failing return values. I am working on the fix.